_posts/2020-06-17-ombulabs-open-source-guidelines.markdown

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title: "OmbuLabs Open Source Guidelines"
+date: 2020-06-17 11:16:00
+categories: ["open-source", "guideline"]
+author: bronzdoc
+---
+
+![goHbayOl3PPzk7yYgEEOzF8q9zqRurIwZexM1ge8.jpg](https://tettra-production.s3.us-west-2.amazonaws.com/0d6efb4f154041e899af17bdcd19c1b5/ce8d5f452da02a88554d80c256162a35/d822b155a4112474fdb7aea5ee22465e/d28647d6b4d6e5842508679707ee21a7/goHbayOl3PPzk7yYgEEOzF8q9zqRurIwZexM1ge8.jpg)
+
+At OmbuLabs we dedicate some of our work hours to open source every week. Also some of us contribute to open source during our spare time.
+
+Here are a few tips to help us keep our OSS contributions organized and encourage people to contribute
+
+<!--more-->
+
+From now on every-time we contribute to an OSS project in working hours, we fork it to the [FastRuby.io organization](https://github.com/fastruby).
+
+This way we can keep track of the contributions we made as a company, also the branches created in the project are accessible to the rest of the team.
+
+As a company we get these benefits from forking our Open Source projects to one of our organizations:
+
+\- We use OSS projects and we want them to be better
+
+\- Our brand becomes associated with these OSS projects
+
+\- Our teammates learn more about these projects
+
+\- Our contributions to be noticed
+
+\- We can use our contributions in the future
+
+\- Sometimes contributions take a long time to be merged into the main OSS project. This way we make sure that we include references to our company’s organization.
+
+\- None of our references will point to a person’s repository. They will point to one of our company’s fork.
+
+As an individual, we get these benefits:
+
+\- Display the user in the PR page
+
+\- Display the contribution in the user's contribution graph
+
+\- Display commits authored by the original user
```

Comment:
Let's stay consistent with the way we display items. Sometimes there are dashes and sometimes dots

![Screen Shot 2020-07-16 at 1 49 09 PM](https://user-images.githubusercontent.com/7442887/87699368-238b9600-c76b-11ea-8977-a5f868ceacac.png)

(https://github.com/fastruby/blog/blob/638549796ee94e0be3b325af066b0cd79b65abc1/_posts/2020-06-17-ombulabs-open-source-guidelines.markdown)